arch/cpu/nrf/doxygen-group.txt

```diff
@@ -0,0 +1,9 @@
+/**
+ * \defgroup nrf The nRF's System-on-Chips
```

Comment:
I'd suggest something like `The nRF5340 and nRF52840 SoCs (using nRF MDK)` to make it more clear which SoCs we support here.

Reply:
Done